app/src/main/java/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -321,19 +323,31 @@ class OfferViewModelImpl(
             insuranceProviderDisplayName,
             paymentMethods
         )
-        val perilItems = OfferItemsBuilder.createPerilItems(data.quoteBundle.quotes)
-        val insurableLimitsItems = OfferItemsBuilder.createInsurableLimits(data.quoteBundle.quotes)
-        val documentItems = OfferItemsBuilder.createDocumentItems(data.quoteBundle.quotes)
+
         val bottomOfferItems = OfferItemsBuilder.createBottomOfferItems(data)
+
         return ViewState.Content(
             topOfferItems = topOfferItems,
-            perils = perilItems,
-            documents = documentItems,
-            insurableLimitsItems = insurableLimitsItems,
+            perils = if (data.quoteBundle.quotes.size == 1) {
+                data.quoteBundle.quotes.first().perils.map { PerilItem.Peril(it) }
+            } else {
+                emptyList()
+            },
+            documents = if (data.quoteBundle.quotes.size == 1) {
+                listOf(DocumentItems.Header(R.string.OFFER_DOCUMENTS_SECTION_TITLE)) +
+                    data.quoteBundle.quotes.first().insuranceTerms
+            } else {
+                emptyList()
+            },
+            insurableLimitsItems = if (data.quoteBundle.quotes.size == 1) {
+                listOf(InsurableLimitItem.Header.Details) +
+                    data.quoteBundle.quotes.first().insurableLimits
+            } else {
+                emptyList()
+            },
```

Comment:
Maybe it was nice that this if size == 1 check was hidden from the VM inside the builder? Not very important but maybe it's a bit nicer :D